build/signedbuild.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         userProvideBuildInfo: msBuildInfo
         msbuildVersion: 16.0
```

Comment:
```suggestion
        msbuildVersion: 17.0
```

Reply:
I tried this in [this build](https://dev.azure.com/mseng/1ES/_build/results?buildId=17711063&view=logs&j=efe68385-d402-5492-a2cf-bc47ec32e784&t=e38751e8-0dc8-5e3c-8d46-e18c39d37211&l=1753), but it doesn't work 😦